### ISSUE TITLE
Add ability to share location with waze

### ISSUE THREAD

**Issue by HarelM** [IsraelHikingMap/Site#1705]
## Feature
As a user I would like to be able to share a public poi, private poi, current location (?) and the beginning (and end?) of a route with waze to allow navigating to this location.

This was somewhat possible before switching to share via os capabilities but apparently removed...

**Comment by zstadler** [comment]
The `https://waze.com/ul?navigate=yes&q=<lat>,<lon>` URL template activates Waze navigation.
For example, https://waze.com/ul?navigate=yes&q=32.187536,35.006483

Source: [How to use Waze Deep Links](https://developers.google.com/waze/deeplinks)

**Comment by HarelM** [comment]
Thanks! (Although this part already already exists in the code, it's just not in use :-))

**Comment by HarelM** [comment]
Current location can be done straight from Waze app, so I don't think it's needed here.
I've added menus to start and end point of a route when route is not in edit mode to allow switching to edit more and to open Waze, if I think there will be more things needed in this menu later on I can always add, but I want to keep the menu not too complicated.
I've added to public's menu the ability to share with Waze too.
I've added to the already overloaded private POI menu...
